### ISSUE TITLE
Regarding epochs 

### ISSUE THREAD

**Issue by khadijarao78** [wocns1457/GAN-based-face-mask-removal#1]
We are working on your code.. Thank you for your hardwork.. But there is a small problem..We are trying to create model it's not saving our mask model in checkpoint folder and also showing steps=zero.. Please help us
![image](https://user-images.githubusercontent.com/106884702/172020643-c7ffa01a-2cdf-4719-9731-94ab5c7bc045.png)

**Comment by wocns1457** [comment]
As a result of executing the training code, it was confirmed that the training progress and the model were successfully saved. I'm not sure exactly where the problem is occurring.
If you still have the same problem, try running the file in the 'src' folder.

Thank you for visiting my repositories.  :)

**Comment by Hudson5697** [comment]
Sir, As i requested , please can you reply as soon as possible we are in trouble, please if u can, please provide us the saved trained model please sir, we request you,,
